content/docs/csidriver/installation/helm/powerstore.md

```diff
@@ -62,18 +62,23 @@ To do this, run the `systemctl enable --now iscsid` command.
 For information about configuring iSCSI, see _Dell PowerStore documentation_ on Dell Support.
 
 
-### Set up the NVMe/TCP Initiator
+### Set up the NVMe Initiator
 
-If you want to use the protocol, set up the NVMe/TCP initiators as follows:
+If you want to use the protocol, set up the NVMe initiators as follows:
 - The driver requires NVMe management command-line interface (nvme-cli) to use configure, edit, view or start the NVMe client and target. The nvme-cli utility provides a command-line and interactive shell option. The NVMe CLI tool is installed in the host using the below command.
 `sudo apt install nvme-cli`
 
+**Requirements for NVMeTCP**
 - Modules including the nvme, nvme_core, nvme_fabrics, and nvme_tcp are required for using NVMe over Fabrics using TCP. Load the NVMe and NVMe-OF Modules using the below commands:
 ```bash
 modprobe nvme
 modprobe nvme_tcp
 ```
 
+**Requirements for NVMeFC**
+- NVMeFC Zoning of the Host Bus Adapters (HBAs) to the Fibre Channel port must be done.
+
```

Comment:
Should we add a note here to not load nvme_tcp in Ubuntu, when using NVMe/FC?

Reply:
Will add that in other PR with all the other details for NVMeFC once the bugfix PR goes in